src/components/NavBar.js

```diff
@@ -16,22 +19,35 @@ function NavBar() {
           </span>
         </Link>
 
-      <div>
-        <button
-          type="button"
-          onClick={() => history.push('/signin')}
-        >
-          Log in
-        </button>
-        <button
-          type="button"
-          onClick={() => history.push('/signup')}
-        >
-          Registreren
-        </button>
-      </div>
-    </nav>
+        <div>
+
+          {auth === false ?
+
+              <button type="button"
+                      onClick={() => history.push ( '/signin' )}
+              >
+                Login
+              </button>:
+
+              <button
+                  type="button"
+                  onClick={() => history.push ( '/signin' )}
+              >
+                Uitloggen
```

Comment:
ik zou kiezen voor "inloggen" en "uitloggen" of "log in" en "log uit". Taalgebruik consistent houden.